### ISSUE TITLE
Nurgle gift that's supposed to give 10% MD gives flat 10 MD instead

### ISSUE THREAD

**Issue by floskan** [chadvandy/cbfm_wh3#848]
Might be the same for other similar effects from the DLC
![20230202232343_1](https://user-images.githubusercontent.com/34427023/216463155-f416d3b4-93db-462d-bd39-9857ebdd3ef2.jpg)
![20230202230103_1](https://user-images.githubusercontent.com/34427023/216463254-d29413f4-7068-44a1-9ccd-a6e0c23d3b66.jpg)
![20230202231921_1](https://user-images.githubusercontent.com/34427023/216463276-2175d378-549b-4fd2-8d3d-fcb7d4a6fdec.jpg)

**Comment by floskan** [comment]
Can we even fix this?

**Comment by mfvreeland** [comment]
We should certainly be able to fix this. It's just an effect bundle.

**Comment by mfvreeland** [comment]
After reviewing this, it would appear that the effect is, in fact, tied to the apparently correct bonus value (melee_defence_mod_mult). This suggests that the bonus value itself is being applied wrong, which is something that takes place on the game code (C++) side of things and is inaccessible to us. 

We could create a new bonus value and script the correct melee defense change, but this bonus value is used in many places, not only in this instance (Cathay and Ogre Kingdoms use it in some places, for example), so we would have to go through and replace it everywhere. This is too much work (with the potential to miss something) for something relatively minor, IMO. 

I suggest we shelve this for now and report the problem with the melee_defence_mod_mult bonus value to CA. They should be able to fix it on the C++ end very easily, and a fix there would naturally propagate to every place where this modifier is used.

**Comment by floskan** [comment]
> 

Do you want me to report this or is it already done?

**Comment by mfvreeland** [comment]
No, because the issue is with the bonus value (what is actually modified in-game), and that can only be fixed in code.
